config/clusters/ubc-eoas/common.values.yaml

```diff
@@ -59,25 +59,27 @@ jupyterhub:
 
   singleuser:
     defaultUrl: /lab
+    memory:
+      # Based on measurements of observed usage
+      guarantee: 1G
+      limit: 1G
```

Comment:
could we increase the limit a touch for both courses? There are a few notebooks that they will use later in the term that might exceed 1G. For EOSC 350, 2G should be fine.

Reply:
@lheagy based on comments from Henryk, these are now set to 4G as they requested that.